docs/en/02_Developer_Guides/01_Templates/02_Common_Variables.md

```diff
@@ -145,6 +145,20 @@ $MetaTags(false)
 <title>$Title - Bob's Fantasy Football</title>
 ```
 
+### Disabling the meta generator tag
+
+The meta generator tag include the current version of `silverstripe/framework`. This version number
+provides aggregate installation numbers to the product team who maintain Silverstripe CMS which is
+used to make informed product decisions.
+
+If it's felt this creates a security concern for your website, then the version portion of the meta
+generator tag can be configured off via
```

Comment:
"configured off" is a bit confusing, maybe something like this might be clearer
```suggestion
generator tag can be disabled via yaml configuration:
```

Reply:
Updated